x/ibc/core/02-client/keeper/grpc_query.go

```diff
@@ -197,3 +197,46 @@ func (q Keeper) ClientParams(c context.Context, _ *types.QueryClientParamsReques
 		Params: &params,
 	}, nil
 }
+
+// UpgradedClientState implements the Query/UpgradedClientState gRPC method
+func (q Keeper) UpgradedClientState(c context.Context, req *types.QueryUpgradedClientStateRequest) (*types.QueryUpgradedClientStateResponse, error) {
```

Comment:
We should also move `UpgradedConsensusState` query to IBC keeper.

Reply:
Sure, I don't think I can remove it from x/upgrade (otherwise proto types would need a version bump). It would just be non-usable in x/upgrade

Reply:
added an issue for this on ibc-go